lib/core/test/unit/Cardano/Pool/DB/Properties.hs

```diff
@@ -935,7 +911,7 @@ prop_listRegisteredPools DBLayer {..} entries =
         monitor $ counterexample $ unlines
             [ "Read from DB: " <> show pools
             ]
-        assert (pools == (view #poolId <$> reverse entries))
+        assert (Set.fromList pools == Set.fromList (view #poolId <$> entries))
```

Comment:
Any reason to remove the check on ordering?

Reply:
> Any reason to remove the check on ordering?

Good question.

The function `listRegisteredPools` doesn't guarantee (currently) to list pools in any particular order, so it doesn't make sense for the property to expect a particular order.

(And as it turns out, the ordering of pools returned by the SQLite implementation of `listRegisteredPools` really is different from the ordering we get back from `Set.toList`. It seems unnecessary to make these orders the same by adding an extra ascending sort to the SQLite implementation.)